glean-core/src/lib.rs

```diff
@@ -613,6 +613,7 @@ impl Glean {
     /// # Errors
     ///
     /// If collecting or writing the ping to disk failed.
+    /// We may still attempt upload if collection succeeded but writing to disk failed.
```

Comment:
IMO this is now confusing here in the docs, because it's under the "errors" section.